src/identify/index.js

```diff
@@ -94,7 +98,7 @@ class IdentifyService {
 
     const pushes = connections.map(async connection => {
       try {
-        const { stream } = await connection.newStream(MULTICODEC_IDENTIFY_PUSH)
+        const { stream } = await connection.newStream(`/${this._libp2p._config.protocolPrefix}/${MULTICODEC_IDENTIFY_PUSH_PROTOCOL_NAME}/${MULTICODEC_IDENTIFY_PUSH_PROTOCOL_VERSION}`)
```

Comment:
Can we create the protocol strings as a property of the identify instance, instead of computing it every time we do a connection identify?

```js
this.identifyProtocolStr = ''
this.identifyPushProtocolStr = ''
```